core/controlplane/config/templates/cloud-config-controller

```diff
@@ -594,17 +605,15 @@ write_files:
          echo 'Failed to create the cluster role named "cluster-admin". It is ok if you are using k8s newer than 1.5'
       fi
 
-      for manifest in {bootstrapped-node,}; do
-          kubectl apply -f "${mfdir}/cluster-roles/$manifest.yaml"
-      done
+      kubectl apply -f "${mfdir}/cluster-roles/node-extensions.yaml"
```

Comment:
I believe it won't make a difference.
The kube-dns and kube-proxy pods will only be scheduled when the first worker node comes up; at that point, all required roles and bindings will already be available.

The kube-proxy running in controllers access the API server in localhost, so it bypasses the auth* mechanism entirely